.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5501
+    "liveServer.settings.port": 5502
```

Comment:
Keep the vscode settings to your local machine. You can add .vscode in the .gitignore file.

Reply:
Since it's already in the main branch, you can fix it later.